identity-server/src/EntityFramework/Services/CorsPolicyService.cs

```diff
@@ -52,7 +52,9 @@ public CorsPolicyService(IConfigurationDbContext dbContext, ILogger<CorsPolicySe
     /// <returns></returns>
     public async Task<bool> IsOriginAllowedAsync(string origin)
     {
+#pragma warning disable CA1308 // this has historically been normalized to lower case
```

Comment:
I was initially concerned, because CA1308's guidance says to not make security decisions based on strings that you have called ToLowerInvariant on, but in this case it is fine and in fact what RFC 3986 says we should do. We're comparing origins (scheme, host and port), and it says:

> the scheme and host are case-insensitive and therefore should be normalized to lowercase.